### PR TITLE
Add Scp1576

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by Thundermaker300** [comment]
Is SCP-1576 permanent?

**Comment by iopietro** [comment]
I don't think so.

File: Exiled.API/Features/Items/Scp1576.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    using System.Text;
+    using System.Threading.Tasks;
+
+    using InventorySystem.Items.Usables;
+    using InventorySystem.Items.Usables.Scp1576;
+
+    /// <summary>
+    /// A wrapper class for <see cref="Scp1576Item"/>.
+    /// </summary>
+    internal class Scp1576 : Usable
```
Comment (iopietro) [inline Exiled.API/Features/Items/Scp1576.cs:22]:
```suggestion
    public class Scp1576 : Usable
```

File: Exiled.API/Features/Items/Scp1576.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Items
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
```
Comment (iopietro) [inline Exiled.API/Features/Items/Scp1576.cs:14]: Remove the usings you don't need.

**Review by iopietro — CHANGES_REQUESTED** [review]

**Comment by Michal78900** [comment]
I wouldn't merge it unless we get a confirmation from NW that's SCP-1576 is permament, which will probably happen after the christmas event ends.

**Comment by iopietro** [comment]
I agree.

**Comment by iopietro** [comment]
I'm merging this, since @louis1706 said this is going to get permanently added.

**Pull request merged**